src/main/resources/application.conf

```diff
@@ -2,7 +2,7 @@ akka {
   loggers = [akka.event.slf4j.Slf4jLogger]
```

Comment:
I guess we don't need this anymore either?

Reply:
It will still be used by the internal cluster actors using the "classic" logging.

Reply:
Yes, but it'll be automatically enabled i thought (https://github.com/akka/akka/blob/master/akka-actor-typed/src/main/resources/reference.conf#L54)

Reply:
Ah, thanks for pointing that out! :)